src/@next/IndexTable/IndexTable.tsx

```diff
@@ -8,14 +8,23 @@ import { Checkbox } from '../Checkbox';
 import { CheckboxProps } from '../Checkbox';
 import { Row } from './components/Row/Row';
 import { StyledIndexTable } from './IndexTableStyle';
+import { LoadingState } from './components/LoadingState';
+
+export interface CustomIndexTableProps
+  extends Omit<IndexTableProps, 'emptySearchTitle'> {
```

Comment:
Any reason why don't we use the existing `emptySearchTitle `?

Reply:
Since we use emptyState there is no need to fill the emptySearchTitle anymore. It's still there but it's optional

Reply:
Oh but then we create a new one with data type `string` after omitting it 🤔 , if we don't need it, agree to omit it, but no need to create a new one in CustomIndexTableProps, also I think you need to re-export `CustomIndexTableProps` as `IndexTableProps` so the user of this component can refer to it

Reply:
I've updated it, please check if it is what you mean